build-api/api.py

```diff
@@ -0,0 +1,10 @@
+from flask import Flask
+
+app = Flask(__name__)
+
+@app.route("/build")
+def build():
+    return "Hello World!"
+
```

Comment:
This PR was about setting up the "scaffolding" for the build API, but not actually triggering the task. That will be handled in the story Cornelius is working on. 

If you want, we can have this do something besides return "Hello World"